ansible/playbooks/roles/redpanda_broker/templates/configs/defaults.j2

```diff
@@ -1,31 +1,49 @@
-{% set address_attribute = 'inventory_hostname' if (advertise_public_ips | d() | bool) else 'private_ip' %}
 cluster:
   rpc_server_tcp_recv_buf: 65536
   enable_rack_awareness: {{ true if rack is defined and rack != -1 else false }}
 node:
-  cluster_id: redpanda
-  organization: redpanda-test
+  organization: "{{ redpanda_organization }}"
+  cluster_id: "{{ redpanda_cluster_id }}"
   redpanda:
+    data_directory: "{{ redpanda_data_directory }}"
     node_id: "{{ id }}"
```

Comment:
Should we remove node_id?

Reply:
I just manually removed node id and did a full reboot of all nodes and the cluster functions as expected.

Reply:
I recommend that we remove all elements of node+id as part of this PR to make adding and removing nodes cleaner.

Reply:
Removing node id is part of #95 . I'm not sure about the edge cases with seed servers yet

Reply:
I've removed `node_id` - seems to work okay